pages/api/issue_card.js

```diff
@@ -15,20 +15,53 @@ export default async function handler(req, res) {
     });
 
     const financialAccount = financialAccounts.data[0];
+    const cardholder = await stripe.issuing.cardholders.retrieve(
+      req.body.cardholderid,
```

Comment:
lets deconstruct the body params at the top so we can quickly see what this handler expects.

```js
const { cardholderid, card_type } = req.body
```

Reply:
Fixed on commit [779b1b0](https://github.com/stripe-samples/treasury-nextjs/pull/19/commits/779b1b01fe53154e3ad6d07714f1c83c51607c17)